### PR TITLE
Cultures/anatolian switch

### PR DESCRIPTION
**Pull request by ghost**
done, both are switched

also some provinces in russia were wrong

### CODE REVIEW

File: CK2Plus_expanded/localisation/000_CK2Plus_expanded_cultures.csv [hunk 1]
```diff
@@ -88,7 +88,7 @@ sephardic;Sephardic;S
 sicilian;Sicilian;Sicilien;Sizilianisch;;Siciliano;;;;;;;;;x
 sicilian_arabic;Siculo-Arabic;Sicilien-Arabe;Sizilian-Arabisch;;Siciliano-�rabe;;;;;;;;;x
 sicilian_greek;Griko;Griko;Griko;;Griko;;;;;;;;;x
-siculo_norman;Siculo-Norman;Italo-Norman;Siculo-Normannen;;Siculo-Normano;;;;;;;;;x
+siculo_norman;Siculo-Norman;Italo-Norman:Siculo-Normannen;;Siculo-Normano;;;;;;;;;x
```
Comment (schwarherz) [inline CK2Plus_expanded/localisation/000_CK2Plus_expanded_cultures.csv]:
```suggestion
siculo_norman;Siculo-Norman;Italo-Norman;Siculo-Normannen;;Siculo-Normano;;;;;;;;;x
```

**Review by schwarherz — CHANGES_REQUESTED** [review]

**Review by schwarherz — APPROVED** [review]

**Comment by schwarherz** [comment]
I fixed the only issue I found. Merging.

**Comment by questionmarkexclamationpoint** [comment]
@daniloy 
While this all looks good, I would like to say that the Russian changes probably should have been made in a separate pull request. This pull request is to fix the turkish/anatolian thing, but you ended up including unrelated changes in it.

We should avoid this. Instead, when you made those changes, you should have:
- Checked out master
- Checked out a new branch for your russian changes
- Made your russian changes
- Pushed them to github and made a separate pull request *for that issue*
- switched back to your anatolian/turkish branch and continued work from there normally

This way we can avoid having huge pull requests with logically unrelated things in them (in this case it wasn't a huge pr, but still).

**Comment by daniloy** [comment]
Sorry, I was greedy

**Comment by questionmarkexclamationpoint** [comment]
Also, we'll need to follow up and fix the dynasty files for `anatolian`.

**Pull request merged**